sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/repository/RepositoryImp.kt

```diff
@@ -25,8 +25,12 @@ class RepositoryImpl @Inject constructor() : Repository {
 
     override suspend fun getColorList() = ColorItem.values().asList()
     override suspend fun getAtomList() = arrayListOf(
-        AtomItem(1, R.string.atoms_text),
-        AtomItem(2, R.string.atoms_buttons),
-        AtomItem(3, R.string.atoms_divider)
+        ComponentMenuItem(1, R.string.atoms_text),
+        ComponentMenuItem(2, R.string.atoms_buttons),
+        ComponentMenuItem(3, R.string.atoms_divider)
+    )
+    override suspend fun getMoleculesList() = arrayListOf(
+        ComponentMenuItem(1, R.string.molecules_inset),
+        ComponentMenuItem(2, R.string.molecules_inset_text)
```

Comment:
Could we give these some nice human readable names rather than just 1 and 2? eg. `const val INSET_TEXT_MOLECULE = 2`